app/hooks/user-hooks.ts

```diff
@@ -95,10 +95,13 @@ const usePriceCache = () => {
 
   const [cachedPrice, setCachedPrice] = React.useState(() => {
     if (initialBtcPrice) {
-      return initialBtcPrice
+      return initialBtcPrice?.formattedAmount
```

Comment:
I think this is the fix @samerbuna.  It was returning a full price object returned from main query instead of a number.